bin/src/command/mod.rs

```diff
@@ -131,11 +106,11 @@ pub enum Success {
     SaveState(usize, String),          // amount of written commands, path of the saved state
     Status(CommandResponseContent),    // Vec<WorkerInfo>
     SubscribeEvent(String),
-    UpgradeMain(i32),         // pid of the new main process
-    UpgradeWorker(u32),       // worker id
-    WorkerKilled(u32),        // worker id
-    WorkerLaunched(u32),      // worker id
-    WorkerOrder(Option<u32>), // worker id
+    UpgradeMain(i32),    // pid of the new main process
+    UpgradeWorker(u32),  // worker id
+    WorkerKilled(u32),   // worker id
+    WorkerLaunched(u32), // worker id
+    WorkerOrder,
```

Comment:
I prefer the name of `WorkerRequest`, I found it more understandable (not sure this word exists XD)